packages/ui/package.json

```diff
@@ -29,6 +29,7 @@
     "react": "16.7.0",
     "react-addons-update": "15.6.2",
     "react-dom": "16.7.0",
+    "react-helmet": "^5.2.1",
```

Comment:
Pin dependency versions please!

Reply:
Thanks, I'll do that now. We could use [that trick](https://github.com/neontribe/www/blob/master/.yarnrc#L7-L8) to make sure that `yarn add`s are exact